px-data-grid.html

```diff
@@ -828,6 +835,32 @@
           }
         }
 
+        /**
+         * Make sure
+         */
+        _columnOrderCleaner() {
+          const columnsRow = this._vaadinGrid._columnTree[this._vaadinGrid._columnTree.length - 1].slice(0);
+          const leftColumns = columnsRow.filter(c => !c.isDataColumn && c.frozen);
+          const rightColumns = columnsRow.filter(c => c.isDataColumn || !c.frozen);
+
+          if (leftColumns && leftColumns.length && rightColumns && rightColumns.length) {
+            const firstRight = rightColumns[0];
+            leftColumns.forEach(c => {
+              this._vaadinGrid.insertBefore(c, firstRight);
+            });
+          }
+        }
+
+        _selectableChanged(selectable) {
+          if (selectable) {
+            setTimeout(() => {
```

Comment:
Can we flush vaadin-grid instead of setTimeout?

Reply:
What would that be? Only flush I found is on test cases.